packages/xstate-inspect/README.md

```diff
@@ -55,3 +55,103 @@ import { inspect } from '@xstate/inspect';
 const service = interpret(someMachine, { devTools: true });
 service.start();
 ```
+
+## Configuration
+
+* `url` *(optional)* - The endpoint that the Inspector sends events to. Default: https://stately.ai/viz?inspect
+* `iframe` *(optional)* - The iframe that loads the provided URL. If iframe is set to `false`, then a new tab is opened instead.
+* `devTools` *(optional)* - Allows custom implementation for lifecycle hooks.
+* `serialize` *(optional)* - A custom serializer for messages sent to the URL endpoint. Useful for sanitizing sensitive information, such as credentials, from leaving your application.
+* `targetWindow` *(optional)* - Provide a preferred `window.open` location that will be used instead. When using this option, you must still provide the `url` value due to security checks in browser APIs.
```

Comment:
The note about `window.open` here makes it sound as if this argument would internally be somehow used with it but it's used instead of it. 

I would also include information here that this makes `iframe` option to be ignored.

Maybe something like this?
```suggestion
* `targetWindow` *(optional)* - Provide a pre-existing window location that will be used instead of opening a new window etc. When using this option, you must still provide the `url` value due to security checks in browser APIs, and the `iframe` option is ignored in such a case.
```

Reply:
Yeah, I like that more as well. Thank you!